### ISSUE TITLE
Tab-complete filenames/paths for --script

### ISSUE THREAD

**Issue by dmiller-nmap** [bonsaiviking/nmap-completion#2]
Right now, --script will tab-complete names of scripts that can be found in Nmap's search path (`$NMAPDIR/scripts/*.nse` and `~/.nmap/scripts/*.nse`) but doesn't work for absolute or relative paths to scripts in other locations.

**Comment by batrick** [comment]
You may want to use `--datadir` if set. The Nmap man page also references an NMAPDATADIR too (although it looks deprecated).

**Comment by dmiller-nmap** [comment]
@batrick Yes, that's a "known bug" in the README. I haven't looked into how to examine more of the command line than the current and immediately preceding argument.
